public/app/features/explore/Logs/utils/uFuzzy.ts

```diff
@@ -10,7 +10,7 @@ const uf = new uFuzzy({
 });
 
 export function fuzzySearch(haystack: string[], query: string, dispatcher: (data: string[][]) => void) {
-  const [idxs, info, order] = uf.search(haystack, query, false, 1e5);
+  const [idxs, info, order] = uf.search(haystack, query, 0, 1e5);
```

Comment:
@gtk-grafana do you actually need the 1e5 ranking limit here? the default is 1e3 and it's probably not a good idea to go higher than 1e4. there's not a huge amount of value to ranking, sorting, and highlighting 100k or even 10k results, but it _is_ slower.

Reply:
No, I do not. These settings confused me so I used the same option I knew was already in use in the metrics explorer. I'll remove